### ISSUE TITLE
How do I generate output from a model?

### ISSUE THREAD

**Issue by Sciss** [dhruvrajan/tensorflow-keras-java#17]
In [this tutorial](https://www.tensorflow.org/tutorials/generative/dcgan), one step tests the untrained generator to produce a noise image:

```python
generator = make_generator_model()
noise = tf.random.normal([1, 100])
generated_image = generator(noise, training=False)
```

I have this:

```scala
val generator: Model[TFloat32] = makeGeneratorModel()
val graph     = new Graph
val tf        = Ops.create(graph)
val noise     = tf.random.randomStandardNormal(tf.constant(Array(1, 100)), classOf[TFloat32])
generator.compile(tf, 
  Optimizers.select(Optimizers.sgd), 
  Losses.select(Losses.sparseCategoricalCrossentropy),
  Seq(Metrics.select(Metrics.accuracy)).asJava)
val generatedImage = ???
```
I don't understand how I can now get values out of the model. Do I need to call something like `fetch` and `run` on the graph, or is there a direct way in Keras? 

Internally, I see I would probably need a session runner, but I see no API that calls it to generate output.

Thanks

**Comment by Sciss** [comment]
I believe it's just `generator.apply()` or `generator.call()`, with the difference in Keras Python that the latter takes additional arguments `testing` and `mask`.
